angular/CHANGELOG.md

```diff
@@ -37,7 +37,7 @@
     unresolved.
 
 *   Support inherited method tear-offs in event listeners. Previously, we only
-    considered methods in the Component class itself.
+    considered methods in the Component class itself. Closes [#506][].
 
 *   Fixed a bug where `[attr.name.if]` did not work on a static `@HostBinding`.
     [Closes #1484][#1484].
```

Comment:
@alorenzen: in general when referring to an issue number, how about **simply writing the issue number in parentheses** at the end of the bullet rather than `Closes #1234`? It would be more concise and amount to less typing.